src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -141,18 +143,37 @@ export class DictionaryConcepts extends Component {
     }));
   }
 
+  checkMembershipStatus(username) {
+    const {
+      match: {
+        params: {
+          type, typeName,
+        },
+      },
+    } = this.props;
+    if (type === 'orgs') {
+      const url = `/${type}/${typeName}/members/${username}/`;
+      this.props.fetchMemberStatus(url);
+    }
+  }
+
   render() {
     const {
       match: {
-        params: { typeName },
+        params: { type, typeName },
       },
       location: { pathname },
       concepts,
       filteredClass,
       filteredSources,
       loading,
+      userIsMember,
     } = this.props;
-    const username = typeName === getUsername();
+    const username = typeName === getUsername() || userIsMember;
```

Comment:
It looks like this variable takes on a boolean. Const `username` = true. Is there a way you can rename this variable to make it more readable such is `userIsMember`?

Reply:
I have made the update.